### ISSUE TITLE
Attempted folder load fails silently

### ISSUE THREAD

**Issue by philrz** [brimdata/zui#2965]
## tl;dr

Dragging a folder of data into Zui has never worked. In the past when a user attempted this, a message popped up with guidance. However this got lost when the Preview & Load feature was introduced.

## Details

Repro is starting with Zui commit 1a79d10 that's associated with the changes in #2834.

A user asked in a [community Slack thread](https://brimdata.slack.com/archives/C010DR0HHMF/p1705085093727719):

> how do we upload a folder of zeek logs to zui

At Zui commit 12404d5 that came right before the introduction of Preview & Load in #2834, if a user tried to drag a folder of input data into Zui, they'd see a message like this guiding them about what to do instead.

![image](https://github.com/brimdata/zui/assets/5934157/d540dea0-ae38-43f8-9abf-5591dbe8b222)

However, since the merge of the changes in #2834, now an Unhandled Runtime Error occurs instead as shown in the following video. Unfortunately in production Zui that error lands in DevTools, so the user basically just sees quiet failure.

https://github.com/brimdata/zui/assets/5934157/dc0daf6e-205d-487c-91c4-f223d530c347

That error:

> Error: Error invoking remote method 'loads.getFileTypes': Error: EISDIR: illegal operation on a directory, read

In terms of what can be done about this, at minimum it certainly would be good to catch this and provide the same guidance as we did in the past. That said, how difficult would it be to just iterate through the folder contents and treat it the same as if the user had done a Select All on the contents of that folder and dragged it in? Even if we only did this with the top-level files inside the folder and didn't try to do any fancy recursion into inner folders, I expect users like this one would have found it useful.
